notebooks/compendium/01.chsh/index.ipynb

```diff
@@ -13,7 +13,7 @@
    "id": "a0bed888",
```

Comment:
<p>I'd avoid calling it "cheating", as it sounds like using entanglement is breaking the rules (i.e. no communication). Same for "thumb on the scale".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/textbook/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>